Изменен стиль кода, изменены переменные

gromkon opened the pull request:
Изменен стиль кода;
Переменная objectCollision (строчка 1) теперь константа (раньше задавалась через let, теперь через const);
Переменная objectTypes (строчка 7) теперь константа (раньше задавалась через let, теперь через const);

gromkon committed:
Изменен стиль кода, изменены переменные

Изменен стиль кода;
Переменная objectCollision (строчка 1) теперь константа (раньше задавалась через let, теперь через const);
Переменная objectTypes (строчка 7) теперь константа (раньше задавалась через let, теперь через const);

AlexsaTasara reviewed (commented):
Перенеси все изменения в один патч.

gromkon committed:
Изменен стиль кода

Переменная tileTypes (строчка 1) теперь константа (раньше задавалась через let, теперь через const);
Убраны коментарии написанные неподдерживаемым шрифтом;

gromkon committed:
Изменен стиль кода, переменные

Переменные stats (бывшая строчка 1, нынешняя 37), subjects (бывшая строчка 7, нынешняя 44) теперь константы (раньше задавались через let, теперь через const);

gromkon committed:
Изменен стиль кода, переменные

Переменные tileW (бывшая строчка 6, нынешняя 11), tileH (бывшая строчка 6, нынешняя 12), теперь константы (раньше задавались через let, теперь через const);
Пауза перенесена на кнопку “P” (бывшая строчка , нынешняя строчка 80)
Переменные keysDown (бывшая строчка 10, нынешняя 14), directions (бывшая строчка 22, нынешняя 27), floorTypes (бывшая строчка 29, нынешняя 34), viewport (бывшая строчка 96, нынешняя 98) теперь константы (раньше задавались через let, теперь через const);
Переменные (старые строчки 261 – 283, нынешние 263 – 285) теперь константы (раньше задавались через let, теперь через const);
Убрана проверка загрузки спрайта ”hero” (бывшие строчки 164 – 167), в следующем обновлении будет добавлена проверка загрузки всех спрайтов;

AlexsaTasara reviewed (approved):
Хорошо.
